azure-pipelines.yml

```diff
@@ -218,16 +218,16 @@ stages:
             TEST_MODE: full
             BITS: 64
             SCIPY_USE_PYTHRAN: 0
-          Python39-64bit-full-ilp64:
-            PYTHON_VERSION: '3.9'
+          Python310-64bit-full-ilp64:
+            PYTHON_VERSION: '3.10'
             PYTHON_ARCH: 'x64'
             TEST_MODE: full
             NPY_USE_BLAS_ILP64: 1
             BITS: 64
             OPENBLAS64_: $(OPENBLAS)
             SCIPY_USE_PYTHRAN: 1
-          Python310-64bit-full:
-            PYTHON_VERSION: '3.10'
+          Python39-64bit-fast:
```

Comment:
did you mean to change the `TEST_MODE` below as well so it matches the name?

Reply:
oops, yes